### ISSUE TITLE
Only works with public activity?

### ISSUE THREAD

**Issue by swrobel** [typicode/ghwn#2]
I have notifications for private repos that aren't showing up in ghwn ... I'm assuming that's because no auth takes place, therefore it can't show private activity, correct?

**Comment by typicode** [comment]
I thought it would, but maybe I'm missing something. Here's the API I'm using:

https://developer.github.com/v3/activity/events/#list-events-that-a-user-has-received

> These are events that you've received by watching repos and following users. If you are authenticated as the given user, you will see private events. Otherwise, you'll only see public events.

**Comment by swrobel** [comment]
Yeah, so it looks like authentication would be required to see private events. I put in my username but it never asked me to authenticate ghwn so I'm guessing it can't access private events.

**Comment by raphink** [comment]
That could be useful, but storing a token will probably require an extension, or using local storage.
